app/vmalert/rule/group_test.go

```diff
@@ -134,6 +134,7 @@ func TestUpdateWith(t *testing.T) {
 	})
 
 	// update recording rule
+	debug := true
```

Comment:
there should be a test that verifies group update if user updated only `.group.debug` field - will it be hot-reloaded?